### DISCUSSION TITLE
Unable to use `request.readBody`. How can I access request body?

### DISCUSSION THREAD

**Discussion by borsemayur2** [alephjs/aleph.js#424] [Q&A] [answered]
I want to access data sent in body of POST request. It was possible with `request.readBody` previously, but it is no longer working. What is the alternative?

**Comment by ije** [comment, marked as answer]
request now is a stand `Request` object, check https://developer.mozilla.org/en-US/docs/Web/API/Request

**Reply by borsemayur2** [reply]
I'm getting `undefined` for `request.body`. What is the right way to use it. I've tried following:
```javascript
await Deno.readAll(request.body)  // Getting error
console.log(request.body)  // undefined
```

**Reply by borsemayur2** [reply]
Worked with `request.json()`. Thanks
